prExplorer/API/Controllers/ProductController.cs

```diff
@@ -1,11 +1,10 @@
 using Application.Products;
-using ApplicationTests;
 using Domain;
 using Microsoft.AspNetCore.Mvc;
 
 namespace API.Controllers
 {
-    public class ProductsController : BaseApiController
+    public class ProductController : BaseApiController
```

Comment:
I guess `Products` was a better name :) You operate on products here, not only on a single product

Reply:
Dawid, we created the project from scratch.
We are here now: https://github.com/BeratARPA/HizliLezzet